openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -56,17 +56,22 @@ This specification uses the terms "Holder", "Issuer", "Verifier", and "Verifiabl
 
 The following aspects are in scope of this interoperability profile:
 
-* Protocol for issuance of the Verifiable Credentials (can be both remote and in-person) (OID4VCI)
-* Protocol for online presentation of Verifiable Credentials (can be both remote and in-person) (OID4VP)
+* Profile of OID4VCI to issue IETF SD-JWT VCs (can be both remote and in-person), including
+  * Wallet Attestation
+* Profile of OID4VP to present IETF SD-JWT VCs
+* Profile of OID4VP over the W3C Digital Credentials API [@w3c.digital_credentials_api] to present 
+  * IETF SD-JWT VCs 
+  * ISO mdocs
 * Protocol for User Authentication by the Wallet at a Verifier (SIOP v2)
-* Wallet Attestation (during Credential issuance)
-* Credential Format (SD-JWT VC)
-* Status Management of the Credentials, including revocation
-* Cryptographic Holder Binding
-* Issuer key resolution
-* Issuer identification (as prerequisite for trust management)
+* Profile of IETF SD-JWT VC that includes the following aspects
+  * Status Management of the Credentials, including revocation
+  * Cryptographic Holder Binding
+  * Issuer key resolution
+  * Issuer identification (as prerequisite for trust management)
 * Crypto Suites
 
+Note that when OID4VP is used, the Wallet and the Verifier can either be remote or in-person.
```

Comment:
```suggestion
Note that when OpenID4VP is used, the Wallet and the Verifier can either be remote or in-person.
```

Reply:
I'm not sure this line provides any benefits

Reply:
given that one of the reader audience is ISO people coming from the proximity world, i think this line is ok